nodejs/week1/avg.js

```diff
@@ -0,0 +1,39 @@
+console.log("Hi I'm avg.js");
+
+const args = process.argv.slice(2);
+
+
+function checkIfNumber(arg) {
+   return !isNaN(parseFloat(arg)) && isFinite(arg);
+};
+
+function getAVG(args) {
+    const sum = args.reduce((acc, arg) => acc + parseFloat(arg), 0);
+    return sum / args.length;
+}
```

Comment:
Nice! The casing of the name of the function is a bit odd though, it should be either `getAvg` or `getAverage` in my opinion.